geoportailv3/static/js/usermanagerservice.js

```diff
@@ -72,9 +72,9 @@ app.UserManager = function($http, loginUrl, logoutUrl,
   this.role = undefined;
 
   /**
-   * @type {number|undefined}
+   * @type {?number}
    */
-  this.roleId = undefined;
+  this.roleId = null;
```

Comment:
I remember some problems with the type checking, but not sure about detail anymore.